scripts/purge-deployments.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euox pipefail
+
+export VALID_NAMESPACES=("hocs-gamma" "hocs-delta" "hocs-qax")
+
+current=$(kubectl config current-context)
+if [[ ${VALID_NAMESPACES[*]} =~ $current ]] ; then
+  echo "Current context is $current"
+else
+  echo "Unauthorised Namespace" >&2
+  exit 1
+fi
+
+echo 'Warning: this action will completely remove all the kubernetes deployments of '${current^^}'. It cannot be undone!'
+
+read -p 'Are you sure you want to remove all the kubernetes deployments (y/n)? ' yes_no
```

Comment:
It's often helpful to put the default in capitals:
```suggestion
read -p 'Are you sure you want to remove all the kubernetes deployments (y/N)? ' yes_no
```